src/hexer/iterinliner.nim

```diff
@@ -392,9 +423,7 @@ proc transformStmt(e: var EContext; c: var Cursor) =
       inc c
       while c.kind notin {EofToken, ParRi}:
         transformStmt(e, c)
-      takeParRi e, c # skipParRi
-    of VarS, LetS, CursorS, ResultS:
```

Comment:
Still entirely correct comment. Handle locals correctly please.

Reply:
I removed them to delegate them to else branch so that each element will be transformed properly. Same as https://github.com/nim-lang/nimony/pull/630